<?xml version="1.0"?>
<launch>

	<arg name="action_server_name"  default="kuka_server" />

	<node name="pour_server_node" pkg="pour_kuka" type="pour_server_node" required="true" output="screen">
		<param name="action_server_name" value="$(arg action_server_name)"/>
  		<param name="action_mode" value="lasa_fixed"/>
  		<param name="simulation" value="false" />
  		<param name="world_frame" value="/world_frame" />
  		<param name="model_dt" value="0.01" />
  		<param name="reachingThreshold" value="0.01" />
  		<param name="orientationThreshold" value="0.05" />
		  <param name="model_base_path" value="$(find lasa_action_planner)../epfl-task-models/PouringModels" />
	</node>
</launch>
